pkg/reader/usdc_reader_solana.go

```diff
@@ -0,0 +1,157 @@
+package reader
```

Comment:
Any tests for Solana? Or would that be covered/tested in Chainlink repository with e2e tests?

Reply:
TBD. I'm not really sure how to properly unit test this since everything would be mocked out. Even with an e2e test a lot of it is mocked out.